src/features/script-load.js

```diff
@@ -30,24 +30,30 @@ systemJSPrototype.createScript = function (url) {
 };
 
 // Auto imports -> script tags can be inlined directly for load phase
-var lastAutoImportUrl, lastAutoImportDeps, lastAutoImportTimeout;
+var lastAutoImportDeps, lastAutoImportTimeout;
 var autoImportCandidates = {};
 var systemRegister = systemJSPrototype.register;
+var inlineScriptCount = 0;
 systemJSPrototype.register = function (deps, declare) {
   if (hasDocument && document.readyState === 'loading' && typeof deps !== 'string') {
     var scripts = document.querySelectorAll('script[src]');
     var lastScript = scripts[scripts.length - 1];
+    var lastAutoImportUrl;
+    lastAutoImportDeps = deps;
     if (lastScript) {
```

Comment:
I think the ideal implementation would use the `src` if it's present and the  `__inline_script__` if it's not. Something like this:

```suggestion
    if (lastScript.src) {
```